doc/developers/advanced_installation.rst

```diff
@@ -490,11 +490,12 @@ Then, add the oneAPI repository to your APT repositories:
     sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
     sudo apt-get update
 
-Install ICC, packaged under the name ``intel-oneapi-icc``:
+Install ICC, packaged under the name
+``intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic``:
 
 .. prompt:: bash $
 
-    sudo apt-get install intel-oneapi-icc
+    sudo apt-get install intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic
```

Comment:
They decided that one (or even two) `cpp` in the name was not explicit enough to indicate that it compiled C++ ?